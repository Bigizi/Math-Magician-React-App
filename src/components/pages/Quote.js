import React from 'react'

function Quote() {
  return (
    <div>
      <h1>Math Magicians!</h1>
      <p>Mathematics is not about numbers, equations, computations or algorithm, It is about understanding. William Paul Thurston</p>
    </div>
  )
}

export default Quote
