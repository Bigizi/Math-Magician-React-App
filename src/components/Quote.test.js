import renderer from 'react-test-renderer';
import Quote from './Quote';

describe('Test Quote component', () => {
  it('Test component should render properly', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
