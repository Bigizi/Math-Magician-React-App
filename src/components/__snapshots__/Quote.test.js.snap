// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Quote component Test component should render properly 1`] = `
<div>
  <h1>
    Math Magicians!
  </h1>
  <p>
    Mathematics is not about numbers, equations, computations or algorithm, It is about understanding. William Paul Thurston
  </p>
</div>
`;
